gtsfm/scene_optimizer.py

```diff
@@ -160,8 +164,8 @@ def create_computation_graph(
             i2Ri1_graph_dict[(i1, i2)] = i2Ri1
             i2Ui1_graph_dict[(i1, i2)] = i2Ui1
             v_corr_idxs_graph_dict[(i1, i2)] = v_corr_idxs
-            two_view_reports_dict[(i1, i2)] = two_view_report
-            two_view_reports_pp_dict[(i1, i2)] = two_view_report_pp
+            for token, report in two_view_reports.items():
+                two_view_reports_dict[token][(i1, i2)] = report
```

Comment:
I do kind of like this "token" idea, but I think it would be cleaner if it was _inside_ the `TwoViewEstimationReport` (or maybe another class that manages all of the reports). Maybe this is ok for as a temporary solution.

Reply:
Yeah in the long term I think we want to restructure `TwoViewReport`, so we can handle it there